src/lf_mask.rs

```diff
@@ -157,45 +168,63 @@ fn mask_edges_inter(
     let t_dim = &dav1d_txfm_dimensions[max_tx as usize];
 
     // See [`decomp_tx`]'s docs for the `txa` arg.
-    let mut txa = Align16([[[[0; 32]; 32]; 2]; 2]);
```

Comment:
I think we can use this old code, just with `0` changed to `MaybeUninit::uninit()`.  It should be safe.